### ISSUE TITLE
Provide nicer displays for GenericPowerModel

### ISSUE THREAD

**Issue by yeesian** [lanl-ansi/PowerModels.jl#460]
They are currently not terminal-friendly for interactive usage.

**Comment by ccoffrin** [comment]
How would this be different than the `print_summary` function?

**Comment by yeesian** [comment]
`print_summary` is for the network data dictionary. I imagine they could be the same, but perhaps users might be interested in seeing model specific information (e.g. number of variables/constraints/etc) based on the formulation of the GenericPowerModel too?

Currently 
```
network_data = PowerModels.parse_file(case3file)
```
looks nice in the terminal (since it renders as a Julia Dict), but
```
pm = build_generic_model(network_data, ACPPowerModel, PowerModels.post_opf)
```
clobbers the terminal screen (when rendering `pm.data`).

**Comment by ccoffrin** [comment]
Good point.  Something more summative would be preferable in that case.
